sdk/storage/Azure.Storage.Blobs/src/Models/BlobInfo.cs

```diff
@@ -8,584 +8,40 @@
-        public byte[] ContentHash => _flattened.ContentHash;
-#pragma warning restore CA1819 // Properties should not return arrays
-
-        /// <summary>
-        /// Details returned when downloading a Blob
-        /// </summary>
-        public BlobDownloadDetails Details { get; private set; }
-
-        /// <summary>
-        /// Creates a new DownloadInfo backed by FlattenedDownloadProperties
-        /// </summary>
-        /// <param name="flattened">The FlattenedDownloadProperties returned with the request</param>
-        internal BlobDownloadInfo(FlattenedDownloadProperties flattened)
-        {
-            _flattened = flattened;
-            Details = new BlobDownloadDetails() {
-                LastModified = flattened.LastModified,
-                Metadata = flattened.Metadata,
-                ContentRange = flattened.ContentRange,
-                ETag = flattened.ETag,
-                ContentEncoding = flattened.ContentEncoding,
-                CacheControl = flattened.CacheControl,
-                ContentDisposition = flattened.ContentDisposition,
-                ContentLanguage = flattened.ContentLanguage,
-                BlobSequenceNumber = flattened.BlobSequenceNumber,
-                CopyCompletedOn = flattened.CopyCompletionTime,
-                CopyStatusDescription = flattened.CopyStatusDescription,
-                CopyId = flattened.CopyId,
-                CopyProgress = flattened.CopyProgress,
-                CopySource = flattened.CopySource,
-                CopyStatus = flattened.CopyStatus,
-                LeaseDuration = flattened.LeaseDuration,
-                LeaseState = flattened.LeaseState,
-                LeaseStatus = flattened.LeaseStatus,
-                AcceptRanges = flattened.AcceptRanges,
-                BlobCommittedBlockCount = flattened.BlobCommittedBlockCount,
-                IsServerEncrypted = flattened.IsServerEncrypted,
-                EncryptionKeySha256 = flattened.EncryptionKeySha256,
-                EncryptionScope = flattened.EncryptionScope,
-                BlobContentHash = flattened.BlobContentHash,
-                TagCount = flattened.TagCount,
-                VersionId = flattened.VersionId,
-                IsSealed = flattened.IsSealed,
-                ObjectReplicationSourceProperties =
-                    flattened.ObjectReplicationRules?.Count > 0
-                    ? BlobExtensions.ParseObjectReplicationIds(flattened.ObjectReplicationRules)
-                    : null,
-                ObjectReplicationDestinationPolicyId = flattened.ObjectReplicationPolicyId,
-                LastAccessed = flattened.LastAccessed
-            };
-        }
-
-        /// <summary>
-        /// Disposes the BlobDownloadInfo by calling Dispose on the underlying Content stream.
-        /// </summary>
-        public void Dispose()
-        {
-            Content?.Dispose();
-            GC.SuppressFinalize(this);
-        }
```

Comment:
where did this go?

Reply:
https://github.com/seanmcc-msft/azure-sdk-for-net/blob/feature/storage/migrateBlobNewGenerator/sdk/storage/Azure.Storage.Blobs/src/Models/BlobDownloadInfo.cs#L100

I moved `BlobDownloadInfo` to its own class.

Reply:
Great. Just making sure we kept `Dispose`.